src/draftv2/setup.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
```

Comment:
Question: what is this new folder and file is for please ? Currently the command runs based on “az aks” extension so is it that the vision is that draftv2 will exist as completely separate extension?